tests/data/search_annotations.json

```diff
@@ -0,0 +1,119 @@
+{
+    "total": 2053,
+    "rows": [
+        {
+            "id": "GMCCHLDcEe-LxNMeSelwtg",
+            "created": "2024-12-02T18:34:42.333087+00:00",
+            "updated": "2024-12-02T18:34:42.333087+00:00",
+            "user": "acct:mdiroberts@hypothes.is",
+            "uri": "https://example.com/",
+            "text": "michael reply",
+            "tags": [],
+            "group": "__world__",
+            "permissions": {
+                "read": [
+                    "group:__world__"
+                ],
+                "admin": [
+                    "acct:mdiroberts@hypothes.is"
```

Comment:
I think we probably want to replace all the annotation and user IDs with fake ones. Just `test_annotation_id_1` and `acct:test_user_1@hypothes.is`, etc.